table/block_based/block_based_table_reader.cc

```diff
@@ -1889,6 +1904,22 @@ Status BlockBasedTable::RetrieveBlock(
         GetMemoryAllocator(rep_->table_options), for_compaction,
         rep_->blocks_definitely_zstd_compressed,
         rep_->table_options.filter_policy.get());
+
+    if (get_context) {
```

Comment:
@anand1976  In ```TableCache::MultiGet```, we first get the ```TableReader``` and then call the ```BlockBasedTable::MultiGet```. In ```TableCahe::GetTableReader``` flow, it creates Index reader which call ```RetrieveBlock``` for index block type with ```get_context = nullptr```. So that won't be logged as of now. Is this ok for the experiment or that should also be recorded?

Reply:
@akankshamahajan15 Its ok to not record it. We don't plan to parallelize file open anytime soon. Even if we want to track it at some point, it should probably be a separate stat.

Reply:
Thanks for clarifying. I have unit test ready. I will upload it by EOD.